proteus/mprans/SW2DCV.py

```diff
@@ -356,7 +356,7 @@ def evaluate(self, t, c):
     def preStep(self, t, firstStep=False):
         if firstStep:
             # Init boundaryIndex
-            assert (self.model.boundaryIndex is None and self.model.normalx is not None, "Check boundaryIndex, normalx and normaly")
+            assert self.model.boundaryIndex is None and self.model.normalx is not None, "Check boundaryIndex, normalx and normaly"
```

Comment:
I was under the impression that python3 syntax required parenthesis around `asserts` and `prints`.

Reply:
Hm, maybe. It kept giving me a warning, so I just took off the parenthesis. Let me double check.

Reply:
It seems like you only need parentheses for a multi-line assert.